server/swap/swap.go

```diff
@@ -1550,7 +1550,7 @@ func (s *Swapper) processRedeem(msg *msgjson.Message, params *msgjson.Redeem, st
 	matchID := match.ID()
 	chain := stepInfo.asset.Backend
 	if !chain.ValidateSecret(params.Secret, cpContract) {
-		log.Errorf("Secret validation failed (match id=%v, maker=%v, secret=%x)",
+		log.Infof("Secret validation failed (match id=%v, maker=%v, secret=%v)",
 			matchID, actor.isMaker, params.Secret)
```

Comment:
%v prints hex and I think this is info. It is the fault of the user if the redemption secret is incorrect.

Reply:
Good call on both accounts.  I fell into the %x-with-a-Stringer pitfall that I like to remind others about.